### PR TITLE
Fixed all new translations not being in "translated" state

### PR DESCRIPTION
**Pull request by xPoppyx**
- Changed translation for "Browse" from file extraction window and "Clear Selection" to a more appropriate one.
- Changed the status of all new translations from "new" to "translated"

### CODE REVIEW

File: Files/MultilingualResources/Files.es-ES.xlf [hunk 1]
```diff
@@ -256,7 +256,7 @@
         </trans-unit>
         <trans-unit id="ExtractFilesDialogBrowseButton.Content" translate="yes" xml:space="preserve">
           <source>Browse</source>
-          <target state="new">Navegar</target>
+          <target state="new">Examinar</target>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.es-ES.xlf:259]: Unless the state is set to `Translated`, the translation won't show up.

Comment (xPoppyx) [reply Files/MultilingualResources/Files.es-ES.xlf:259]: oh I didn't know that

**Comment by xPoppyx** [comment]
I forgot to mention that "Drives" was not in the file to translate

**Comment by yaichenbaum** [comment]
@xPoppyx Thank you, that should be fixed with #556.

**Pull request merged**